src/screens/NestedInfoScreen.tsx

```diff
@@ -42,6 +57,20 @@ const ListHeaderComponent = ({ loading, html }: { loading: boolean; html?: strin
       <Wrapper>
         {/* @ts-expect-error HtmlView uses memo in js, which is not inferred correctly */}
         <HtmlView html={html} />
+        {!!subQuery && !!subQuery.routeName && subQuery.params && (
+          <Button
+            title={subQuery.buttonTitle}
+            onPress={() =>
+              navigation.navigate({
+                name: subQuery.routeName,
+                params: {
+                  ...subQuery.params,
+                  title: subQuery.params?.title
+                }
+              })
+            }
+          />
+        )}
```

Comment:
how come that you have added this one button here? was this possible to be copied from elsewhere in the code base to be used here?
i have compared with `HtmlScreen` where a similar situation is already implemented.
what about reusing or copying that? this would also enable multiple buttons if necessary:

```
{!!subQuery && !!subQuery.routeName && (!!subQuery.webUrl || subQuery.params) && (
  <Button
    title={subQuery.buttonTitle || `${title} öffnen`}
    onPress={() => navigate()}
  />
)}

{!!subQuery &&
  !!subQuery.buttons &&
  subQuery.buttons.map((button, index) => (
    <Button
      key={`${index}-${button.webUrl}`}
      title={button.buttonTitle || `${title} öffnen`}
      onPress={() =>
        navigate({
          routeName: button.routeName,
          webUrl: button.webUrl
        })
      }
    />
  ))}
```

Reply:
I worked with only one button logic because we don't need more buttons, but I can change that.

Yes, it is the same as in `HtmlScreen`, but navigation is not returned from a function.

Reply:
with more buttons, we can realize one button. with only implementing one button we are not prepared for requests to have more buttons. as we already have the code for more buttons, let's use it here as well please.

Reply:
i will create a component to avoid code repetition.

Reply:
a new component has been created to prevent code repetition. New `MultiButtonWithSubQuery` component has been added to `HtmlScreen` and `NestedInfoScreen`

https://github.com/ikuseiGmbH/smart-village-app-app/pull/697/commits/28605c4223fcf6a7db415de28d7ac01fbc86ee38